crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -219,13 +229,21 @@ impl AnalyzeEcmascriptModuleResultBuilder {
                 }
             }
         }
+        let mut source_map = None;
+        for map in self.source_maps {
+            if let Some(map) = &*map.await? {
+                source_map = Some(map.resolve().await?);
+                break;
+            }
+        }
```

Comment:
Why are we collecting multiple Source Maps, when we only use the first one?

And: Should be use the last one instead?